src/test/scala/org/broadinstitute/hail/methods/ImputeSexSuite.scala

```diff
@@ -35,16 +35,20 @@ class ImputeSexSuite extends SparkSuite {
 
   object Spec extends Properties("ImputeSex") {
 
+    val plinkSafeBiallelicVDS = VariantSampleMatrix.gen(sc, VSMSubgen.plinkSafeBiallelic)
+      .resize(1000)
+      .map(vds => vds.filterVariants { case (v, va, gs) => v.isAutosomalOrPseudoAutosomal && v.contig.toUpperCase != "X" && v.contig.toUpperCase != "Y" })
+      .filter(vds => vds.nVariants > 2 && vds.nSamples >= 2)
+
     property("hail generates same results as PLINK v1.9") =
-      forAll(VariantSampleMatrix.gen[Genotype](sc, VSMSubgen.random)) { case (vds: VariantSampleMatrix[Genotype]) =>
+      forAll(plinkSafeBiallelicVDS) { case (vds: VariantSampleMatrix[Genotype]) =>
 
         var s = State(sc, sqlContext).copy(vds = vds.copy(rdd =
           vds.rdd.map { case (v, (va, gs)) => (v.copy(contig = "X"), (va, gs)) }
             .toOrderedRDD))
 
-        s = SplitMulti.run(s)
         s = VariantQC.run(s)
-        s = FilterVariantsExpr.run(s, Array("--keep", "-c", "va.qc.AC > 0"))
+        s = FilterVariantsExpr.run(s, Array("--keep", "-c", "va.qc.AC > 1 && va.qc.AF >= 1e-8 && va.qc.nCalled * 2 - va.qc.AC > 1 && va.qc.AF <= 1 - 1e-8"))
```

Comment:
The inbreeding coefficient code in PLINK has a cutoff of MAC > 1 and MAF >= 1e-8 (even though the comment line says it's supposed to remove monomorphic). The reason for checking both sides is there is no concept of a reference allele in PLINK. It coerces whichever of the two alleles are most common to the reference allele. Hail does not do that, so we needed to check both sides for the test to work. Same for the InbreedingCoefficientSuite.